### ISSUE TITLE
Database name UI input is ignored

### ISSUE THREAD

**Issue by dimitri-furman** [TPC-Council/HammerDB#753]
**Describe the bug**
When I enter a non-default (other than `tpcc`) database name in the Options dialogs (either for building a schema or for driver script) for MSSQL TPROC-C, it is ignored. The loaded script has `tpcc` as the database name. Changing the name in the script directly is a workaround.

**To Reproduce**
1. Go to the GUI dialog for configuring the schema build or the driver script.
2. Enter a non-default database name.
3. Review the generated script and observe that the database name is `tpcc`.

**Expected behavior**
The entered database name is used in the script.

**Screenshots**
![image](https://github.com/user-attachments/assets/dbe7f658-ff8b-4f6b-b35f-32014d67bd50)

![image](https://github.com/user-attachments/assets/11c226ce-155d-4775-8bbd-65ff14393642)

**HammerDB Version (please complete the following information):**
 - Version: 4.11
 - Build: Release download

**HammerDB Interface (please complete the following information):**
 - UI: GUI

**Operating System (please complete the following information):**
 - Client OS: Windows Server 2022

**Database Server (please complete the following information):**
 - Database name: MSSQL
 - Database Release Version: N/A

**Database Client (please complete the following information):**
 - Database client name: ODBC Driver 18 for SQL Server

**Comment by sm-shaw** [comment]
Many thanks for raising the issue. Yes, this is a duplicate of this issue https://github.com/TPC-Council/HammerDB/issues/721 and is a confirmed bug and is fixed by https://github.com/TPC-Council/HammerDB/pull/722.
The release version including this fix, HammerDB v4.12 has been staged ready for release for 3 weeks now and is only waiting for TPC-Council steering committee approval.  So ideally this should be available very, very soon. 

In the meantime, there is an easy single character workaround modifying the source source. 

The issue was caused by https://github.com/TPC-Council/HammerDB/pull/678 (by changing the settings dialogs to 2 tabs) and fix is a 1 character fix in file src/mssqlserver/mssqlsopt.tcl as shown below changing **f1 for Setting field to c1 for connection field** as database is shown on the connection tab.

```
set tpccfields [ dict create tpcc {mssqls_dbase {.tpc.f1.e6 get} ...
set tpccfields [ dict create tpcc {mssqls_dbase {.tpc.c1.e6 get} ...

set tpchfields [ dict create tpch {mssqls_tpch_dbase {.mssqlstpch.f1.e6 get} ...
set tpchfields [ dict create tpch {mssqls_tpch_dbase {.mssqlstpch.c1.e6 get} ...
```

But as noted going forward this is already done in the v4.12 release.

**Comment by dimitri-furman** [comment]
Thanks, @sm-shaw, I missed the older issue.
